executable.go

```diff
@@ -13,6 +13,9 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var minBackoff *time.Duration
```

Comment:
Definitely shouldn't be using globals here. These should be passed into the function.